permissions/plugin-maven-artifact-choicelistprovider.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "maven-artifact-choicelistprovider"
+paths:
+- "org/jenkins-ci/plugins/maven-artifact-choicelistprovider"
```

Comment:
@daniel-beck : Do you know, why i cant see my plugin on the jenkins http://jenkins.ci.cloudbees.com ?